### PR TITLE
liveness probe & readyness probe

### PR DESCRIPTION
**Pull request by saikatharryc**
#1

### CODE REVIEW

**Comment by saikatharryc** [comment]
@4nte  need your attention here, resolved conflict.

File: charts/volantmq/values.yaml [hunk 1]
```diff
@@ -71,6 +71,28 @@ testFramework:
...
+readinessProbeInitialDelay: 30
+
+# whether configure livenessProbe on controller pod
+enableLivenessProbe: false
+
+# How long to wait (in seconds) before checking the liveness probe
+livenessProbeInitialDelay: 30
+
+# How long to wait before timeout (in seconds) when checking controller liveness
+livenessProbeTimeout: 1
```
Comment (4nte) [inline charts/volantmq/values.yaml]: Could you make these values grouped like this?
```suggestion
readinessProbe:
  enable: true
  interval: 60
  timeout: 3
  initialDelay: 30
  
livenessProbe:
  enable: false
  initialDelay: 30
  timeout: 1
```
:ok_hand:

Comment (saikatharryc) [reply charts/volantmq/values.yaml]: Done, can you have a look once?

**Review by 4nte — APPROVED** [review]
Looks good

**Pull request merged**